### DISCUSSION TITLE
Changing primary email in google provider

### DISCUSSION THREAD

**Discussion by vaishnavsm** [outline/outline#3987] [Self Hosting] [answered]
Hey folks,

We internally changed the primary email of users in Google workspace. When migrating old users to new users, they are denied logins. Either deleting the account, or deleting the user_authentication in the db, both allow them to log back in, but with a _new_ profile, not the existing one. Just changing the `email` property in the `users` table does not let them log in either. I assume some encoded data depends on the old email?

Is there any way to migrate the email of an existing profile?
If not, would this be a bug, and how can I help fix this?

On version 0.65.2, self hosted on docker.

Thanks a bunch!

**Comment by vgwidt** [comment, marked as answer]
It might be related to https://github.com/outline/outline/issues/3438.  You could check the logs to see if the error matches.

**Reply by vaishnavsm** [reply]
Oh yeah, that's it. Thanks a bunch @vgwidt ! Any idea when 0.66 will launch? just to know if I should patch it myself or wait for the release.

**Reply by vgwidt** [reply]
Not really sure, but it seems like about 2 months pass before each release, so I'd guess 0.66 might be around the end of September.
